.github/workflows/release-new.yml

```diff
@@ -0,0 +1,104 @@
+# Copyright 2021 ZUP IT SERVICOS EM TECNOLOGIA E INOVACAO SA
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+name: release-new
```

Comment:
Can we just use "release" here?

Reply:
it will be conflicting with the existing release [file](https://github.com/ZupIT/horusec/blob/main/.github/workflows/release.yml), i intend to test this on newer versions and on success erase the existing file and change the name